templates/apps/create.html

```diff
@@ -247,14 +259,24 @@ <h6> {{ vals.meta_title }}</h6>
                         {% endif %}
 
                         {% if subval.type == "number" %}
-                        <label class="form-label">{{ subval.title }}</label>
-                        {% if subkey == "port"%}
-                        <span class="bi bi-question-circle" style="color: #989da0" data-bs-toggle="tooltip" title="" data-bs-placement="right" data-bs-original-title="Port that the docker container exposes and the application runs on.">
-                        </span>
-                        <input type="number" id="{{ subkey }}" name="{{ key }}.{{ subkey }}" placeholder="{{ subval.default }}" value="" class="form-control" required>
-                         <div class="invalid-feedback">
-                            Please add a valid port!
-                        </div>
+                        {% if subkey == "port" or subkey == "userid" %}
+                            {% if subkey == "port" %}
+                                <label class="form-label">{{ subval.title }}</label>
+                                <span class="bi bi-question-circle" style="color: #989da0" data-bs-toggle="tooltip" title="" data-bs-placement="right" data-bs-original-title="Port that the docker container exposes and the application runs on.">
+                                </span>
+                                <input type="number" id="{{ subkey }}" name="{{ key }}.{{ subkey }}" placeholder="{{ subval.default }}" value="" class="form-control" required>
+                                <div class="invalid-feedback">
+                                    Please add a valid port!
+                                </div>
+                            {% elif subkey == "userid" and request.user.is_superuser %}
```

Comment:
Only superusers can set user id?

Reply:
Yes, in my opinion this makes sense.  Although we have limited the uid range, it would still be good for us to know exactly what uids are being used and why.